index.html

```diff
@@ -5,13 +5,13 @@
   <meta charset="UTF-8" />
```

Comment:
Please add a link to the corresponding line of code in the Pull Request which required changes.